### PR TITLE
Hello-Microverse-project

### PR DESCRIPTION
**Pull request by DhuruvSingh**
Added Linter

Added HTML File
Added CSS File
Added Test File

### CODE REVIEW

File: node_modules/.bin/autoprefixer.ps1 [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env pwsh
+$basedir=Split-Path $MyInvocation.MyCommand.Definition -Parent
+
+$exe=""
+if ($PSVersionTable.PSVersion -lt "6.0" -or $IsWindows) {
+  # Fix case when both the Windows and Linux builds of Node
+  # are installed in the same directory
+  $exe=".exe"
+}
```
Comment (youmari) [inline node_modules/.bin/autoprefixer.ps1]: you had pushed node_modules then you added it to `.gitignore` Always try to add the  ignored files or folder to the `.gitignore` file before you push, so try to remove the node_modules from the repo it makes it unprofessional and confusing with a lot of files

File: Test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (youmari) [inline Test.md]: you had pushed the file before you added it to `.gitignore ` file  Always try to add the  ignored files or folder to the `.gitignore` file before you push, so try to remove the `Test.md`  from the repo

File: README.md [hunk 3]
```diff
@@ -1 +1,53 @@
...
+# Hello Microverse Project
+
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+## Live Demo
```
Comment (youmari) [inline README.md]: Try to remove the empty headers it makes your `README.md` file not good looking and unprofessional

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.github/workflows/linters.yml
+node_modules/
+Test.md/
```
Comment (youmari) [inline .gitignore]: Try to remove the slash from `Test.md` file, we add a slash for folders

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @DhuruvSingh , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

# HIghlights
✔️ No linters error
✔️ Great implementing the header h1 and CSS style 

## Suggested changes

 _Check the comments under the review._
- [x] Also I see that you commit not descriptive enough please next time try to follow these [rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: node_modules/.bin/autoprefixer.ps1 (hunk 1 above)
Comment (DhuruvSingh) [reply node_modules/.bin/autoprefixer.ps1]: Sure i will remember that and i will also remove node_module from the repo

File: Test.md (hunk 2 above)
Comment (DhuruvSingh) [reply Test.md]: Yeah sure

File: README.md (hunk 3 above)
Comment (DhuruvSingh) [reply README.md]: ok

File: .gitignore (hunk 4 above)
Comment (DhuruvSingh) [reply .gitignore]: oh thanks

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+.github/workflows/linters.yml
```
Comment (phelian23) [inline .gitignore]: - [x] [Optional] This file should not be ignored. Kindly remove it from the .gitignore file

**Review by phelian23 — APPROVED** [review]
Hi @DhuruvSingh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 5 above)
Comment (DhuruvSingh) [reply .gitignore]: Yes sure.

I put it because my mentor told me that you can put this file to git ignore to prevent any other issues.

**Comment by DhuruvSingh** [comment]
Thank you so much @youmari And @phelian23 for your valuable Feedback.

**Pull request merged**